workshops/collaborative_sketch/README.md

```diff
@@ -48,23 +54,49 @@ And we'll add script tags for Firebase, p5.js, jQuery, and our own `main.js` fil
 
 ### Setting up the Firebase App
 
-Much like we did in the last workshop, we'll need to create an app within Firebase. In this project, we'll be using Firebase to store the points of what we're drawing on our canvas.
+Firebase is a real-time database. This means that all the information we send to Firebase gets updated across everything connected to that database.
+
+In this project, we'll be using Firebase to store the points of what we're drawing on our canvas. We'll need to create an app within Firebase.
+
+Now we'll set up the Firebase side of our project. Go to [Firebase](https://firebase.google.com/) and sign up for an account.
+
+Now that we're at our dashboard, we'll create our app. Let's head over to our [Firebase dashboard](https://console.firebase.google.com/) and create a new project, which we will name `collaborative-sketch`.
+
+Let's click on "Add Firebase to your web app." We'll copy _part_ of the code snippet and paste it into our `main.js`:
+
+```js
+var config = {
+  apiKey: "AIxaSyGsAkHke9lXEU_97a8rYpMn7gOH3eWDxrM",
+  authDomain: "collaborative-sketch.firebaseapp.com",
+  databaseURL: "https://collaborative-sketch.firebaseio.com",
+  storageBucket: "collaborative-sketch.appspot.com",
+};
+firebase.initializeApp(config);
+```
+
```

Comment:
I think they should be reminded that this is something that's unique for everyone, because pasting the one from this workshop will be wrong. Something like "Your code snippet will look something like this, but please do not copy paste the one from the workshop."